server.js

```diff
@@ -1210,6 +1210,37 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// Package Control integration.
+camp.route(/^\/packagecontrol\/(.*)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var userRepo = match[1];  // eg, `Package%20Control`.
```

Comment:
Could you add the corresponding badge (`https://img.shields.io/packagecontrol/dt/Package%20Control.svg`) to `try.html`, please?